scripts/lib/common.sh

```
@@ -0,0 +1,177 @@
+    export GIT_BRANCH=${GIT_BRANCH:-main}
+
+    # Installation directory
+    export INSTALL_DIR=${BISQ_SUPPORT_INSTALL_DIR:-/opt/bisq-support}
+    export DOCKER_DIR="${DOCKER_DIR:-$INSTALL_DIR/docker}"
+    export COMPOSE_FILE="${COMPOSE_FILE:-docker-compose.yml}"
+}
+
+# Trap for cleanup on exit
+setup_cleanup_trap() {
+    local cleanup_function="$1"
+    trap "$cleanup_function" EXIT INT TERM
```

Shellcheck (0.11.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)